geomstats/_backend/numpy/__init__.py

```diff
@@ -201,7 +201,7 @@ def assignment_by_sum(x, values, indices, axis=0):
     return x_new
 
 
-def gather(x, indices, axis=0):
+def get_slice(x, indices):
```

Comment:
`gather` was referring to:
https://www.tensorflow.org/api_docs/python/tf/gather
(which does not exist in np),

but I agree that get_slice maybe more clear!

Reply:
I also changed the name because of this: the Tensorflow version relies on `gather_nd` and not `gather` (maybe there was a way to achieve that with it, but I couldn't find any easy one)